packages/api/src/middleware/adminMiddleware.ts

```diff
@@ -0,0 +1,33 @@
+import { NextFunction, Request, Response } from 'express';
+import { Admin } from '@hangar/database';
+
+export const adminMiddleware = async (req: Request, res: Response, next: NextFunction) => {
+  const adminQuery = { user: req.user?.id };
+
+  let userAdmin;
+  try {
+    userAdmin = await req.entityManager.findOne(Admin, adminQuery);
+  } catch {
+    res.sendStatus(500);
+    return;
+  }
+
+  if (userAdmin) {
+    res.sendStatus(200);
+  } else {
+    // Admin does not exist in the database
+    res.sendStatus(403);
+    return;
+  }
```

Comment:
By sending a status here you effectively end the request, which means calling `next` won't result in a handler that can actually respond with anything 😬 
```suggestion
  if (!userAdmin) {
    // Admin does not exist in the database
    res.sendStatus(403);
    return;
  }
```